networkx/drawing/layout.py

```diff
@@ -617,3 +619,178 @@ def flatten(l):
         else:
             yield el
 
+
+def kamada_kawai(G, pos=None, K=1.0, L=0.1, iterations=None,
+                 tolerance=1, D=0.0, weight='weight'):
+    """Position nodes using Kamada-Kawai force-directed algorithm.
+
+    Parameters
+    ----------
+    G : NetworkX graph
+
+    pos : dict or None  optional (default=None)
```

Comment:
Just write `dict (default=None)`, the "optional" is implied by having a default value.